packages/back-end/src/models/AuditModel.ts

```diff
@@ -34,7 +37,120 @@ const auditSchema = new mongoose.Schema({
   dateCreated: Date,
 });
 
-// type AuditDocument = mongoose.Document & AuditInterface;
+type AuditDocument = mongoose.Document & AuditInterface;
 
-// TODO: don't export and add toInterface() method https://github.com/growthbook/growthbook/issues/1300
-export const AuditModel = mongoose.model<AuditInterface>("Audit", auditSchema);
+const AuditModel = mongoose.model<AuditInterface>("Audit", auditSchema);
+
+/**
+ * Convert the Mongo document to an AuditInterface, omitting Mongo default fields __v, _id
+ * @param doc
+ */
+const toInterface = (doc: AuditDocument): AuditInterface => {
+  return omit(doc.toJSON<AuditDocument>(), ["__v", "_id"]);
+};
+
+export async function insertAudit(data: Partial<AuditInterface>) {
```

Comment:
We are trying to be explicit with the return types, can you add the function return type?